curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/add-inline-styling-to-elements.md

```diff
@@ -25,13 +25,15 @@ Add the `style()` method to the code in the editor to make all the displayed tex
 Your `h2` elements should have a `font-family` of `verdana`.
 
 ```js
-assert($('h2').css('font-family') == 'verdana');
+const headingTwo = document.querySelector('h2');
+const fontFamily = window.getComputedStyle(headingTwo)['font-family'];
```

Comment:
```suggestion
assert.isNotNull(headingTwo);
const fontFamily = window.getComputedStyle(headingTwo)['font-family'];
```

doing this should avoid the `TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.` error